scripts/cal_acc.py

```diff
@@ -13,27 +13,32 @@
 #  limitations under the License.
 import sys
 import argparse
-from evaluator import evaluate
+from ns_evaluator import LMEvalParser
+from accuracy import cli_evaluate
 
 if __name__ == "__main__":
     parser = argparse.ArgumentParser(description="Evaluate accuracy for a model")
     parser.add_argument('--model_name', type=str, default="~/Llama-2-7b-chat-hf")
     parser.add_argument('--tasks', type=str, default="lambada_openai")
-    parser.add_argument('--model_format', type=str, default="runtime")
+    parser.add_argument('--model_format', type=str, default="neural_speed")
```

Comment:
I think we don't need this, right? Since we only measure our `NS` format model?

Reply:
removed